src/components/Calculator.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
- [ ] Calculator.js should have App.js imported.

Reply:
#### 2nd Doubt 

I don't understand the purpose or function of importing App.js into Calculator.js to then import Calculator.js into App.js.

Could you please expand on your argument? :confused:

Reply:
I didn't say you import calculator.js to app.js
Please make sure you have imported app.js to calculator.js for future purposes.

Reply:
:warning:  DISCLAIMER TO THE NEW CODE REVIEVWER :warning:
```
I didn't say you import calculator.js to app.js
Please make sure you have imported app.js to calculator.js for future purposes.
```
:rotating_light:  This last requirement of the last code reviewer was a mistake on his part, so please omit it :rotating_light: 

:heavy_check_mark: 1st Request : " What I meant is please change app.js code to class-based."